src/Entity/Feed.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\FeedRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Blameable\Traits\BlameableEntity;
+use Gedmo\SoftDeleteable\Traits\SoftDeleteableEntity;
+use Gedmo\Timestampable\Traits\TimestampableEntity;
+
+#[ORM\Entity(repositoryClass: FeedRepository::class)]
+class Feed
+{
+    use TimestampableEntity;
+    use BlameableEntity;
+    use SoftDeleteableEntity;
+
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private ?int $id = null;
+
+    #[ORM\Column(length: 255)]
+    private ?string $name = null;
+
+    #[ORM\Column]
+    private array $configuration = [];
+
+    #[ORM\Column]
+    private ?\DateTimeImmutable $lastRead = null;
+
+    #[ORM\Column]
+    private ?bool $enabled = null;
+
+    #[ORM\ManyToOne(inversedBy: 'feeds')]
+    private ?User $user = null;
+
+    #[ORM\OneToMany(mappedBy: 'feed', targetEntity: Event::class)]
+    private Collection $events;
```

Comment:
> Associations are marked as Lazy by default, which means the whole collection object for an association is populated the first time its accessed. 

Since doctrine 2.1... we could use extra lazy, but not sure that will do anything here: https://www.doctrine-project.org/projects/doctrine-orm/en/2.6/tutorials/extra-lazy-associations.html#extra-lazy-associations